dashboard/app/controllers/api/v1/pd/workshop_enrollments_controller.rb

```diff
@@ -69,7 +70,7 @@ def create
           sign_up_url: url_for('/users/sign_up'),
           cancel_url: url_for(action: :cancel, controller: '/pd/workshop_enrollment', code: enrollment.code)
         }
-      else
+      rescue ActiveRecord::ActiveRecordError
```

Comment:
can you be more specific? this has lots of subclasses. if you are intentionally trying to rescue more than one error type, I would still recommend explicitly listing them here, so we don't accidentally squash an error we weren't expecting.

Reply:
Good call –– that will also help with Turner's ask of having more helpful error messages